app/merginapi.h

```diff
@@ -39,6 +39,7 @@ typedef QList<std::shared_ptr<MerginProject>> ProjectList;
 
 class MerginApi: public QObject {
     Q_OBJECT
+    Q_PROPERTY(QString username READ username WRITE setUsername NOTIFY usernameChanged)
 public:
     explicit MerginApi(const QString& root, const QString& dataDir, QByteArray token, QObject* parent = nullptr );
```

Comment:
we should get rid of passing token to MerginApi constructor. Better if clients of MerginApi create it without token and then call authorize()